src/recording/recordingmanagerbase.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <QDateTime>
+#include <QList>
+#include <QObject>
+#include <QString>
+
+class RecordingManagerBase : public QObject {
```

Comment:
Why does this class exist?

Reply:
To reduce duplication. I posted about that some time ago in my Zulip topic.
https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/Macros.2FSaved.20Hotcue.20Routines.2F.22Serato.20Flip.22/near/201474356

Reply:
I don't think I would have bothered factoring this out, but I suppose it doesn't hurt.